configs/top_down/darkpose/coco/res101_coco_256x192_dark.py

```diff
@@ -96,7 +96,7 @@
         ]),
 ]
 
-valid_pipeline = [
+val_pipeline = [
```

Comment:
did you searched through the whole codebase for `valid_pipeline`?

Reply:
Yes.